demos/tests/cases.py

```diff
@@ -377,7 +377,7 @@ def single_option_cases(key, *args):
             ModelArg('vehicle-license-plate-detection-barrier-0106')),
     )),
 
-    # TODO: object_detection_demo_yolov3_async: no models.lst
+       # TODO: object_detection_demo_yolov3_async: no models.lst
```

Comment:
Please remove this change.

Reply:
done

Reply:
No, don't remove the line, remove the change.

Reply:
I do not understand your request. Roll  back from 68c020b  to 4ae21df ?

Reply:
Just restore these lines to the state they were originally in.

Reply:
originally when and in which branch? in current develop this comment on L196

`PythonDemo(subdirectory='object_detection_demo_ssd_async'` on L380

`PythonDemo(subdirectory='segmentation_demo',` in L390

My just after it

Reply:
> originally when and in which branch?

In the last commit from `develop` that you merged in. Currently, that is 6367f580.

Reply:
OK, I took cases.py from #6367f58 and have added my demo to it.